design-patterns/gc-ministers/_base.scss

```diff
@@ -0,0 +1 @@
+/* no custom CSS for ministers block */
```

Comment:
Please delete this file if it is not needed.

Reply:
Deleted